front/src/app/components/forms/commons/AgencySelector.tsx

```diff
@@ -187,6 +192,7 @@ export const AgencySelector = ({
           disabled={agencyKindOptions.length === 0 || shouldLockToPeAgencies}
           nativeSelectProps={{
             ...agencyKindField,
+            ...register(agencyKindFieldName),
```

Comment:
c'est necessaire le mémo ? on ne peut pas faire ça :

```suggestion
            ...register(agencyKindField.name),
```

Reply:
Nope sinon on a un problème au niveau du typage